### PR TITLE
Use `abs` from utils instead of the global

### LINKED ISSUE #3068: Use of `abs` bazel function removes compatibility from bazel <7
This was added here https://github.com/bazelbuild/rules_rust/commit/8e962a16d9f4a243e739320015b5d7f5c357cc68

Seems like `abs` is not available in 6.x

### PR DESCRIPTION
**Pull request by avdv**
Fixes #3068

### CODE REVIEW

File: cargo/private/cargo_utils.bzl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """Utility functions for the cargo rules"""
 
 load("//rust/platform:triple_mappings.bzl", "system_to_binary_ext")
+load("//rust/private:utils.bzl", "abs")
```
Comment (avdv) [inline cargo/private/cargo_utils.bzl:4]: Seems I cannot depend on the private utils.bzl file from here. Should I just add the code here?

Comment (avdv) [reply cargo/private/cargo_utils.bzl]: I have just added a copy of the `abs` function to `cargo/private/cargo_utils.bzl`.

File: cargo/private/cargo_utils.bzl [hunk 2]
```diff
@@ -2,6 +2,20 @@
 
 load("//rust/platform:triple_mappings.bzl", "system_to_binary_ext")
 
+# TODO: remove after dropping support for Bazel < 7 when `abs` is a global
```
Comment (illicitonion) [inline cargo/private/cargo_utils.bzl:5]: Our minimum supported bazel version is already 7.3.1 - I'm happy to merge this if it's convenient for folks, but just FYI we don't officially support Bazel 6 :)

**Review by illicitonion — APPROVED** [review]
Works for me as a convenience for folks, but FYI we already don't support Bazel 6 :)

**Comment by avdv** [comment]
> Works for me as a convenience for folks, but FYI we already don't support Bazel 6 :)

OK, fair enough. Is this documented somewhere or just common knowledge?

Thank you for merging. :)

**Comment by illicitonion** [comment]
Our support policy can be found at https://bazelbuild.github.io/rules_rust/#supported-bazel-versions

**Pull request merged**